src/microbe_stage/Microbe.cs

```diff
@@ -1355,6 +1355,19 @@ public override void _IntegrateForces(PhysicsDirectBodyState physicsState)
         physicsState.Transform = GetNewPhysicsRotation(physicsState.Transform);
     }
 
+    /// <summary>
+    ///   Unbinds all colony members
+    /// </summary>
+    public void UnbindAll()
```

Comment:
I think this comment and name is incorrect, because this only removes this cell from the colony, meaning that if this isn't the colony leader then this doesn't "unbind all"